shared/common-adapters/floating-box.desktop.js

```diff
@@ -5,7 +5,7 @@ import {Box} from './box'
 import {ModalPositionRelative} from './relative-popup-hoc.desktop'
 import type {Props} from './floating-box'
 
-const StyleOnlyBox = (props: any) => <Box style={props.style} children={props.children} />
+const StyleOnlyBox = (props: any) => <Box children={props.children} />
```

Comment:
It turns out the style was being applied twice. Not used anywhere until this point.